client/js/models/Home.js

```diff
@@ -0,0 +1,14 @@
+module.exports = { ...require('./__proto__.js'),
+	data: {
+		imageMeta1: [
+			{filename: 'Jam.jpg', alt: 'Jam', date: 'Nov. 12', caption: 'Wild Berry Jam'},
+			{filename: 'Beets.jpg', alt: 'Beets', date: 'Oct. 4', caption: 'We Got The Beets'},
+			{filename: 'Squash.jpg', alt: 'Squash', date: 'Oct. 13', caption: "It's Time For Squash"}
+		],
+		imageMeta2: [
+			{filename: 'Vine.jpg', alt: 'Vines', date: 'August. 19', caption: 'Chicken In The Vine'},
+			{filename: 'Carrots.jpg', alt: 'Carrots', date: 'July. 9', caption: 'Cute Misfit Carrots'},
+			{filename: 'Kale.jpg', alt: 'Kale', date: 'June. 7', caption: 'First Kale Of 2017'}
+		]
+	}
+}
```

Comment:
I would rearrange this into an array of objects. A good rule of thumb is that any time you're sticking numbers on the end of a name, you can probably reorganize. Remove `imageMeta1` and `imageMeta2`, make `data` an array, and then just have all of the file objects be inside that.